src/Common/tests/System/Net/Capability.Security.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+
+namespace System.Net.Test.Common
+{
+    public static partial class Capability
+    {
+        private const string CARootThumbprint = "3B279AD43D6DD459268D3F3A3D72DAAD4BF4D9C6";
```

Comment:
Added a comment containing the Subject and the location of the p7b file containing the file.